jobs/JEVS_GLOBAL_ENS_PLOTS

```diff
@@ -1,79 +1,130 @@
-
-########################################
-# Preliminary data setup step
-########################################
+#!/bin/bash
+#  JOB card created for running METPlus-based Global enesmble verifications
+#   By Binbin Zhou, IMSG@EMC/NCEP
+#   Feb 2, 2022
+#
+#   August 14 2024     Ho-Chun Huang    Merged with JEVS_GLOBAL_ENS_CHEM_GRID2OBS_PLOTS
+#                                       in #!/bin/bash
+#
+#######################################################
 set -x
+##################################################
+### SENDCOM  - Copy Files From TMPDIR to $COMOUT
+### SENDDBN  - Issue DBNet Client Calls 
+### SENDECF  - Flag Events on ecFLOW
+### SENDMAIL - Send email if file is missing
+##################################################
+export SENDCOM=${SENDCOM:-YES}
+export SENDDBN=${SENDDBN:-YES}
+export SENDECF=${SENDECF:-YES}
+export SENDMAIL=${SENDMAIL:-NO}
 
+#######################################################
+# SET SHELL PROCESSING VARIABLES
+#######################################################
+date
+export PS4='$SECONDS + '
+
+export vhr=${vhr:-00}
+export cycle=${cycle:-t${vhr}z}
 
 ##########################################################
 # obtain unique LSF id (jobid) and make temp directories
 ##########################################################
 export pid=$$
 export DATA=${DATA:-${DATAROOT:?}/${jobid:?}}
-rm -rf $DATA
+if [ "${RUN}" != "chem" ]; then
+    rm -rf $DATA
+fi
 mkdir -p $DATA
 cd $DATA
 
-############################################
-# Run setpdy and initialize PDY variables
-############################################
-export vhr=${vhr:-00}
-export cycle=${cycle:-t${vhr}z}
-setpdy.sh
-. $DATA/PDY
+export envir=${envir:-prod}
 
-if [ $RUN = atmos ] || [ $RUN = wmo ] || [ $RUN = headline ] ; then
-	  export VDATE=${VDATE:-$PDYm2}
-fi
+# Define COMIN/COMOUT variables:
 
-########################################
-# Set data directives
-########################################
-export SENDCOM=${SENDCOM:-YES}
-export SENDDBN=${SENDDBN:-YES}
-export SENDECF=${SENDECF:-YES}
-export SENDMAIL=${SENDMAIL:-NO}
+echo ${COMPATH}
 
-########################################
-# Define COMIN/COMOUT variables
-########################################
-export COMIN=${COMIN:-$(compath.py $envir/com/$NET/$evs_ver)}
-export EVSIN=${EVSIN:-$COMIN/stats/$COMPONENT/$MODELNAME.$VDATE}
-export EVSINapcp24mean=${EVSINapcp24mean:-$COMIN/stats/$COMPONENT/$RUN.$VDATE/apcp24_mean}
-export COMOUT=${COMOUT:-$(compath.py -o $NET/${evs_ver})}/$STEP/$COMPONENT/$RUN.$VDATE
-export DATA_IN=${DATA_IN:-$DATAROOT}
-mkdir -p $COMOUT
+export NET=${NET:-evs}
+export STEP=${STEP:-plots}
+export COMPONENT=${COMPONENT:-global_ens}
+export RUN=${RUN:-atmos}
+export VERIF_CASE=${VERIF_CASE:-grid2obs}
+export MODELNAME=${MODELNAME:-gefs}
+export modsys=${modsys:-gefs}
+export mod_ver=${mod_ver:-${gefs_ver}}
+
+# Run setpdy and initialize PDY variables
+if [ "${RUN}" == "chem" ]; then   ## can be combined later, template for wave if different
```

Comment:
Both of these are doing the same thing since the current working directory is DATA. Can we make it so it is

`setpdy.sh`
`. ./PDY`